resources/views/site/blocks/paragraph.blade.php

```diff
@@ -81,4 +81,10 @@
     libxml_clear_errors();
     libxml_use_internal_errors($oldInternalErrors);
 @endphp
-{!! $content !!}
+@if (isset($hasWrapper) && $hasWrapper)
+<div class="paragraph-wrapper">
+    {!! $content !!}
+</div>
+@else
+    {!! $content !!}
+@endif
```

Comment:
@nikhiltri This isn't exactly what we discussed, but I think it's an elegant solution. Apparently, we can pass arbitrary data into blocks (https://github.com/area17/twill/blob/2.x/src/Models/Behaviors/HasBlocks.php#L23), so if I render the contact section blocks like so:
```php
{!! $item->renderNamedBlocks('contact', data: ['hasWrapper' => true]) !!}
```
then I can include the paragraph wrapper dynamically!

Reply:
Nice!